### PR TITLE
macro to read TOF DCS entries

### PR DESCRIPTION
**Pull request by chiarazampolli**

### CODE REVIEW

**Comment by chiarazampolli** [comment]
This should be a trivial change, just the addition of a macro. @shahor02 , I think @noferini  would not complain if you approve it, since it changes nothing in the TOF code.

**Comment by chiarazampolli** [comment]
Hello @TimoWilken , @ktf , 

I am confused... I ran clang-format and nothing was meant to be changed:

```
[O2/latest] ~/SOFT/alibuild/ali-o2/O2 $> git clang-format
no modified files to format
[O2/latest] ~/SOFT/alibuild/ali-o2/O2 $>
```

I thought that the CI and what one gets from alibuild was the same... 
Chiara

**Review by shahor02 — DISMISSED** [review]
@chiarazampolli do you have locally the same clang-format 11 as running in CI? This should be the reason of the discrepancy.

**Comment by chiarazampolli** [comment]
It was just a trivial mistake, all is fine. Locally I could reproduce it. I noticed after I added the comment (now deleted). 
Sorry for the noise.
And thanks @shahor02 for approving!

**Comment by chiarazampolli** [comment]
Error in dataflow seems unrelated.

File: Detectors/TOF/calibration/macros/readTOFDCSentries.C [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#include "CCDB/CcdbApi.h"
+#include "DetectorsDCS/AliasExpander.h"
+#include "DetectorsDCS/DeliveryType.h"
+#include "DetectorsDCS/DataPointIdentifier.h"
+#include "TOFCalibration/TOFDCSProcessor.h"
+
+#include <string>
+#include <unordered_map>
+#include <chrono>
+#include <bitset>
```
Comment (shahor02) [inline Detectors/TOF/calibration/macros/readTOFDCSentries.C]: Could you wrap these includes in ``#if !defined(__CLING__) || defined(__ROOTCLING__)`` / ``#endif``

Comment (chiarazampolli) [reply Detectors/TOF/calibration/macros/readTOFDCSentries.C]: Done

Comment (chiarazampolli) [reply Detectors/TOF/calibration/macros/readTOFDCSentries.C]: Hello @shahor02 , 

Was it not all includes that were needed in the ifdef?

Comment (shahor02) [reply Detectors/TOF/calibration/macros/readTOFDCSentries.C]: No, the ``DetectorsDCS/AliasExpander.h`` is not a class known to ROOTCLING via dictionary but just a declaration of simple functions, should be visible also to ROOTCLING.
Actually, my comment related to other macro which failed in the prev. CI test, I just put it in a wrong place...

Comment (chiarazampolli) [reply Detectors/TOF/calibration/macros/readTOFDCSentries.C]: I don't understand why the "read" macro failed, if the "make" macro did not when I committed it the first time. I thought they should be the same in this respect, and the "make" macro was not failing before, why should the "read" do? So I treated them in the same way. Unless there are some specific "includes" in the "read" one.

Anyway, now the "make" does not contain the "ifdef" anymore, only the "read".

Comment (chiarazampolli) [reply Detectors/TOF/calibration/macros/readTOFDCSentries.C]: I also cleaned up the includes now.

**Pull request merged**